### PR TITLE
Only one <html> element allowed in document

### PR DESCRIPTION
**Pull request by shallow-beach**
make unique rule explicit

While this can be inferred from the previous sentences, I think saying it explicitly couldn't hurt.

### CODE REVIEW

**Comment by shallow-beach** [comment]
Apparently not true - multiple `<html>` tags seem to render out of one _file_ in up-to-date Firefox. Not sure if this is out-of-spec behavior, or if it would be helpful to note somewhere that a document _is defined by_ a single `<html>` tag and its contents? (I.e. that a file can contain multiple documents? See [body](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body)'s use of 'document'.)

**Comment by shallow-beach** [comment]
Sorry, newbie editing docs. Realized Firefox actually merges the content of duplicate tags under the hood. So I think my edit stands.

**Comment by shallow-beach** [comment]
Similar request #35288 was merged with main.

File: files/en-us/web/html/element/html/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ browser-compat: html.elements.html
 
 {{HTMLSidebar}}
 
-The **`<html>`** [HTML](/en-US/docs/Web/HTML) element represents the root (top-level element) of an HTML document, so it is also referred to as the _root element_. All other elements must be descendants of this element.
+The **`<html>`** [HTML](/en-US/docs/Web/HTML) element represents the root (top-level element) of an HTML document, so it is also referred to as the _root element_. All other elements must be descendants of this element. There can be only one **`<html>`** element in a document.
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/html/index.md:10]:
```suggestion
The **`<html>`** [HTML](/en-US/docs/Web/HTML) element represents the root (top-level element) of an HTML document, so it is also referred to as the _root element_. All other elements must be descendants of this element. There can be only one `<html>` element in a document.
```

**Review by Josh-Cena — APPROVED** [review]
Thanks

**Pull request merged**